cmd/kubeadm/app/cmd/init.go

```diff
@@ -112,12 +113,30 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 		"etcd client key file. Note: The path must be in /etc/ssl/certs",
 	)
 	cmd.PersistentFlags().MarkDeprecated("external-etcd-keyfile", "this flag will be removed when componentconfig exists")
+	cmd.PersistentFlags().BoolVar(
+		&skipChecks, "skip-checks", false,
+		"skip checks normally run before modifying the system",
+	)
```

Comment:
@luxas mentioned this as well and I am not taking this lightly. As an user, I think _pre-flight_ is less clear about the intents of this flag, which is to perform checks. I suggest `skip-preflight-checks`.
That said, all log entries related to pre-flight should include a reference to _pre-flight check {failed,succeeded}_.